model.py

```diff
@@ -8948,16 +8963,37 @@ def setting(self, key):
             _db, key, self
         )
 
-    def get(self, key):
+    def get(self, key, default=None):
```

Comment:
I'm not convinced this method is necessary, but it wasn't working before and this should fix it.